fullSimulation.py

```diff
@@ -1,21 +1,25 @@
-from mpi4py                 import MPI
+from pygyro.diagnostics.diagnostic_collector import DiagnosticCollector
+from pygyro.utilities.savingTools import setupSave
+from pygyro.poisson.poisson_solver import DensityFinder, QuasiNeutralitySolver
+from pygyro.advection.advection import FluxSurfaceAdvection, VParallelAdvection, PoloidalAdvection, ParallelGradient
+from pygyro.initialisation.setups import setupCylindricalGrid, setupFromFile
+from pygyro.model.grid import Grid
+from pygyro.model.layout import LayoutSwapper, getLayoutHandler
```

Comment:
@dominikbell Normally all python built-in imports should come first. Do you know why this was re-ordered?

Reply:
I am not sure, I reordered them now and it didn't change it upon formatting ..